server/routes/v1/tokenDiscovery.js

```diff
@@ -133,10 +133,13 @@ async function cacheTokensList() {
 // Starts the caching process for bigquery
 function startCaching() {
   // Only run if on mainnet (the tokens page doesn't exist on devnet/testnet)
-  if (process.env.REACT_APP_ENVIRONMENT !== 'mainnet') {
-    return
-  }
+  // TODO: uncomment below code when token discovery is used again
```

Comment:
We use token discovery still for token detail page.

Reply:
oh right, uncommenting...